### ISSUE TITLE
`customize` error

### ISSUE THREAD

**Issue by emacsomancer** [rainstormstudio/nerd-icons.el#24]
`M-x customize` seems to fail to execute, with error:

````
nerd-icons-octicon: Unable to find icon with name ‘nf-oct-settings’ in icon set ‘octicon’Error during redisplay: (eval (doom-modeline-segment--buffer-info)) signaled (error "Unable to find icon with name ‘nf-oct-settings’ in icon set ‘octicon’")
Error during redisplay: (eval (doom-modeline-segment--buffer-info)) signaled (error "Unable to find icon with name ‘nf-oct-settings’ in icon set ‘octicon’")
````

[connected to: https://github.com/seagle0128/doom-modeline/issues/630 ]

**Comment by il-santo** [comment]
I didn't notice what this issue was about. :)
Just posted an issue (#28) with the resolution for this.

**Comment by seagle0128** [comment]
Closed by 9e8cece99bf6a55783c436d1650f8f6d1a86d1c3.
